modules/lightning_features/lightning_media/tests/src/FunctionalJavascript/MediaReferenceFieldTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\lightning_media\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\JavascriptTestBase;
+
+/**
+ * @group lightning
+ * @group lightning_media
+ */
+class MediaReferenceFieldTest extends JavascriptTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $profile = 'lightning';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $strictConfigSchema = FALSE;
```

Comment:
I believe A todo with the d.org issue pointing to it would be enough for this PR